docs/docusaurus/docusaurus.config.js

```diff
@@ -224,20 +224,24 @@ module.exports = {
       '@docusaurus/preset-classic',
       {
         docs: {
-          includeCurrentVersion: false,
+          includeCurrentVersion: true,
           sidebarPath: require.resolve('./sidebars.js'),
           // Note: remarkCodeImport is included to handle earlier versions with line number references (e.g. v0.14.13)
           remarkPlugins: [remarkNamedSnippets, remarkCodeImport],
           // versions needs to map major.minor -> major.minor.patch for display purposes. Update the patch as needed.
           versions: {
+            current: {
+              label: '1.0 Prerelease',
```

Comment:
Does this need to match our actual version specifier?
If so that would look like one of these
```
X.YaN   # Alpha release
X.YbN   # Beta release
X.YrcN  # Release Candidate
X.Y     # Final release
```

https://packaging.python.org/en/latest/specifications/version-specifiers/#pre-releases

Reply:
That's a good question. I'm not sure it really needs to be anything specific, but it might make sense to match the version specifier. But for example, docusaurus uses `Canary 🚧`. Looks like pydantic uses `dev` in their dropdown. I've viewed it as more of a branding question than anything. My plan has been to touch base with devrel and updating to what they prefer before releasing. But please throw a changes requested on here if you feel strongly we should follow the convention you showed!